requirements.txt

```diff
@@ -10,6 +10,8 @@ ipython
 jupyter_client
 mypy; python_version >= '3.6'
 nbconvert
+pyjwt>=2.4.0; python_version >= '3.6'
```

Comment:
I don't love this conditional dependency here, but if we don't have a version of `pyjwt` installed for older versions of python, we (unsurprisingly) get import errors. 

The feature is only supported for python > 3.6, so that the pyjwt dependency in older versions is really a placeholder. 

If somehow the user was able to run the feature using python 3.5, they would encounter failures because some of the implementation details in pyjwt changed between version ~1.7 (the last supported version for 3.5) and the 2.4.0/2.5.0 pyjwt versions. But this should be a non-issue because we only support this feature for python >= 3.6.

Just providing some context here - if someone has a cleaner solution I'd be happy to implement it.